src/Admin.js

```diff
@@ -151,4 +153,9 @@ Admin.propTypes = {
     initialState: PropTypes.object,
 };
 
-export default Admin;
+export default withContext(
+    {
+        authClient: PropTypes.func,
```

Comment:
I should probably add some warnings in every components that can take a function as child when `authClient` is null.